### ISSUE TITLE
[Feature Ask] Input Widget supports multiple option selection or checkbox

### ISSUE THREAD

**Issue by ShilinHe** [Chainlit/chainlit#611]
As title says, sometimes in the UI configuration, we might want to select multiple options at the same time. It is not the current "Select" feature since it only allows picking one instead of many from a list. Checkbox might be another way to support. Thanks

**Comment by willydouhard** [comment]
Like a multi select or a combo box if I understand correctly?

**Comment by anugandhi11** [comment]
Yes, that would be great if we can add multi select feature for select elements

**Comment by ShilinHe** [comment]
> Like a multi select or a combo box if I understand correctly?

Yes, that would be very helpful in cases such as selecting multiple plugins.

**Comment by remobor1sam** [comment]
Is there any available Chainlit version for this fearure?

**Comment by chenrq2005** [comment]
If Chainlit can have this feature, that would be great!
